core/controllers/reader.py

```diff
@@ -346,9 +411,14 @@ class ReaderFeedbackHandler(base.BaseHandler):
 
     @require_playable
     def post(self, exploration_id):
-        """Handles POST requests."""
+        """Handles POST requests.
+        It records a thread_id of the feedback from a user.
```

Comment:
The function shouldn't be described as "it", better would be "Records the thread_id of feedback from a learner"